### ISSUE TITLE
5$: missing comma between array literal entries should report error

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#916]
The following code is missing some commas, but quick-lint-js does not report any diagnostics:
```javascript
let x = [[1 2 3]];
```
